_pages/pre-work.md

```diff
@@ -142,10 +142,10 @@ discuss. This is subject to a few conditions:
 and add a few extra rules and restrictions of our own:
 
 - Choose from one of these three exercises, and only complete a single exercise:
-    - https://homework.adhoc.team/fetch/
-    - https://homework.adhoc.team/hhbuilder/
-    - https://homework.adhoc.team/slcsp/
-    - https://homework.adhoc.team/noclist/
+    - [https://homework.adhoc.team/fetch/](https://homework.adhoc.team/fetch/)
+    - [https://homework.adhoc.team/hhbuilder/](https://homework.adhoc.team/hhbuilder/)
+    - [https://homework.adhoc.team/slcsp/](https://homework.adhoc.team/slcsp/)
+    - [https://homework.adhoc.team/noclist/](https://homework.adhoc.team/noclist/)
```

Comment:
This is minor and I'm OK with it if we don't, but if we're linking these, what if we don't expose the link? Instead, we could just have [noclist](full url) and be a bit clearer.

Also, should we mention that we're directing them to the adhoc site?

Reply:
Well, I guess the way it is makes the external link obvious at least :).  I think I'll leave as is for now.